source/demos/README.md

```diff
@@ -0,0 +1 @@
+# demos
```

Comment:
It's too bad this empty as I think this is almost as important as the code smile. In my opinion, here we need an additional Dockerfile with run script and instructions how to use it and what to expect from the demo

Reply:
Oh yeah definitely this README need to contain an explanation of each demo. We can do a specific Dockefile as well. Maybe just add a stage to the existing one and a script where you can specify which script to run. Let me try that.

Reply:
True,  something like `FROM project-dependencies as demos` might work

Reply:
I'm just brainstorming here...Maybe a seperate Dockerfile that can run all (future) demos would be better anyway. Such a Dockerfile could use the install script and just install the demos on top. I think this is favorable because as a user I would check how I could install these demos on my computer and if I see our 'development' Dockerfile, I'd be quite confused. The Dockerfile with just the install script would then be almost like instructions how to install control libraries locally.
Moreover, we could even publish such a docker image so the users can just pull it from here and use it to run the demos without the need to build it.

Reply:
@domire8 Dockerfile and run script done